pkg/github/github.go

```diff
@@ -61,6 +63,9 @@ const releasesURLTemplate = "https://api.github.com/repos/%s/releases"
 // e.g. https://api.github.com/repos/cyberark/secretless-broker/compare/v1.5.2...HEAD
 const compareURLTemplate = "https://api.github.com/repos/%s/compare/%s...%s"
 
+// e.g. https://api.github.com/repos/cyberark/secretless-broker/branches/branchName
+const branchesURLTemplate = "https://api.github.com/repos/%s/branches/%s"
```

Comment:
There's 3 variables now with the same prefix - we should extract that common string into a var

Reply:
heyo I tried to start doing that and you dinged me for it - can we put that off to future work? this PR already has value, and this change will unduly delay it.